lib/splitclient-rb/split_config.rb

```diff
@@ -29,7 +29,7 @@ def initialize(opts = {})
       @events_uri = (opts[:events_uri] || SplitConfig.default_events_uri).chomp('/')
       @mode = opts[:mode] || SplitConfig.default_mode
       @redis_url = opts[:redis_url] || SplitConfig.default_redis_url
-      @redis_namespace = opts[:redis_namespace] || SplitConfig.default_redis_namespace
+      @redis_namespace = opts[:redis_namespace] ? "#{opts[:redis_namespace]}/#{SplitConfig.default_redis_namespace}" : SplitConfig.default_redis_namespace
```

Comment:
My understanding according to the spec is: `userPrefix.SPLITIO.blablabla` and i don't see that here.